graphql/schema/types/task.graphql

```diff
@@ -159,6 +159,7 @@ type TaskEndDetail {
   timeoutType: String
   type: String!
   traceID: String
+  diskDevices: [String]
```

Comment:
I'm not sure what our formal guidelines are around GraphQL types, but I like typing lists like the following:
```suggestion
  diskDevices: [String!]!
```
because it prevents `diskDevices` and any of its entries from being `null`, which is generally easier to handle on the frontend. 😃

Reply:
done!